db/test/test_stats.py

```diff
@@ -206,23 +202,6 @@ def test_write_and_get_statistics_data(self):
         ]
         self.assertEqual(list(expected_data), list(data))
 
-    def test_write_stats_error(self):
-        stats1 = {"lowlevel-lossy": 10, "lowlevel-lossy-unique": 6,
-                  "lowlevel-lossless": 15,
-                  "lowlevel-lossless-unique": 10,
-                  "lowlevel-total": 25, "lowlevel-total-unique": "not-a-number"}
```

Comment:
todo: restore a test that ensures that we can render a row with a missing key or non-numeric value